Tchap/Modules/Room/DataSources/RoomDataSource.m

```diff
@@ -110,7 +110,7 @@ - (void)finalizeInitialization
 
     // Sadly, we need to make sure we have fetched all room members from the HS
     // to be able to display read receipts
-    if (!self.isPeeking && ![self.mxSession.store hasLoadedAllRoomMembersForRoom:self.roomId])
+    if (![self.mxSession.store hasLoadedAllRoomMembersForRoom:self.roomId])
```

Comment:
Why did you remove `isPeeking` check here ?

Reply:
It's a mistake. I'll fix it